airflow/gcp/operators/campaign_manager.py

```diff
@@ -0,0 +1,382 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This module contains Google CampaignManager operators.
+"""
+import tempfile
+from typing import Any, Dict, Optional
+import uuid
+
+from googleapiclient import http
+
+from airflow import AirflowException
+from airflow.gcp.hooks.gcs import GoogleCloudStorageHook
+from airflow.gcp.hooks.campaign_manager import GoogleCampaignManagerHook
+from airflow.models.baseoperator import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class GoogleCampaignManagerDeleteReportOperator(BaseOperator):
+    """
+    Deletes a report by its ID.
+
+    .. seealso::
+        Check official API docs:
+        https://developers.google.com/doubleclick-advertisers/v3.3/reports/delete
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GoogleCampaignManagerDeleteReportOperator`
+
+    :param profile_id: The DFA user profile ID.
+    :type profile_id: str
+    :param report_id: The ID of the report.
+    :type report_id: str
+    :param api_version: The version of the api that will be requested for example 'v3'.
+    :type api_version: str
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any. For this to work, the service accountmaking the
+        request must have  domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    template_fields = (
+        "profile_id",
+        "report_id",
```

Comment:
Should add report_name to list of template fields.
```suggestion
        "report_name",
        "report_id",
```